addons/l10n_din5008/data/report_layout.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="report_layout_din5008" model="report.layout">
+            <field name="view_id" ref="l10n_din5008.external_layout_din5008"/>
+            <field name="image">/l10n_din5008/static/img/preview_din.png</field>
+            <field name="pdf">/l10n_din5008/static/pdf/preview_din.pdf</field>
```

Comment:
The files `preview_din.{png,pdf}` are not up to date anymore. If they're still used somewhere? Otherwise, remove them.

Reply:
They are indeed outdated and not used anymore.
So I removed them :ok_hand: